account_statement_import_online/models/online_bank_statement_provider.py

```diff
@@ -69,7 +69,10 @@ class OnlineBankStatementProvider(models.Model):
         string="Update Schedule",
         compute="_compute_update_schedule",
     )
-    last_successful_run = fields.Datetime(string="Last successful pull")
+    last_successful_run = fields.Datetime(
+        string="Last successful pull",
+        readonly=True,
```

Comment:
Why doing this readonly? What if I want to change it manually?

Reply:
@pedrobaeza Why would you change this manually? Is is literally the last time the job last run.

Reply:
OK, if it's not used for any sync related thing.

Reply:
@pedrobaeza As for what it is fixing: suppose the server running Odoo is not always active (for upgrade or other maintenance etc.). It means that the next_run which is supposed to be updated at regular intervals (example every 4 hours), is not always updated and will get further and further back in time. As the `_scheduled_pull` method only retrieves data until the next_run date, you will be getting data back with ever greater delays.  This makes sure that at the end of each run, when the next_run is increased by the run interval, the next_run will be in the future.

Reply:
And why not simply change the fetch to get from last_run to now?

Reply:
Simply running unti now would also solve the problem I outlined. Adjusting the next_run has the same effect with the added advantage of correctly showing when the next run will be.

Reply:
Well, I always prefer the simplest formula, but as you prefer.